### PR TITLE
performance-operator: prepare for branch 4.7

### PR DESCRIPTION
**Pull request by ffromani**
Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fedepaol** [comment]
@fromanirh there are failures in the basic lanes, they normally suggest the fix.
In this case, you should run `make jobs` to generate the jobs files.

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml [hunk 1]
```diff
@@ -76,7 +76,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tag_specification:
-  name: "4.7"
+  name: "4.8"
```
Comment (fedepaol) [inline ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml:79]: I _think_ all 4.8 imagestreams are not available yet.
I left this as 4.7 in cnf-features-deploy.

**Comment by cynepco3hahue** [comment]
@fromanirh Can we please divide this PR into two PRs one for master and one for 4.7 because currently, we do not have CI at all for the 4.7 branch.

**Comment by fromanirh** [comment]
> @fromanirh Can we please divide this PR into two PRs one for master and one for 4.7 because currently, we do not have CI at all for the 4.7 branch.

yeah, better start with 4.7

**Comment by fromanirh** [comment]
obsoleted by https://github.com/openshift/release/pull/15043

**Pull request closed without merging**